### PR TITLE
Adding banner for wix installer

### PR DESCRIPTION
**Pull request by karanbirsingh**
This PR adds a banner image for the wix installer as referenced in issue #292 - here is a preview of what it looks like: 

image

### CODE REVIEW

**Comment by ferBonnin** [comment]
@karanbirsingh the heart icon looks slightly pixeleated to me, is it just the screenshot?

**Comment by DaveTryon** [comment]
We need to check this on Win7. The system installer has problems with images that use transparency

**Comment by DaveTryon** [comment]
I meant to leave a comment, not close this. Reopening

File: src/MSI/Product.wxs [hunk 1]
```diff
@@ -22,6 +22,7 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
     <Property Id="WIXUI_INSTALLDIR" Value="INSTALLFOLDER"/>
 
     <WixVariable Id="WixUIDialogBmp" Value="Resources\DialogBackground.png" />
+    <WixVariable Id="WixUIBannerBmp" Value="Resources\WixDialogBanner.png" />
```
Comment (jalkire) [inline src/MSI/Product.wxs]: Can we not use the existing BrandIcon_256x.png file? Would be good to keep down the number of copies of our icons.

Comment (karanbirsingh) [reply src/MSI/Product.wxs]: Dimensions don't match unfortunately - please see #292 for more info. I am looking into scaling options w/ Fer and Chelsea ATM

**Comment by karanbirsingh** [comment]
@ferBonnin I have updated the screenshot with Chelsea's 3x image.

**Comment by ferBonnin** [comment]
@karanbirsingh looks better! :) thanks!

File: src/AccessibilityInsights/AccessibilityInsights.csproj [hunk 2]
```diff
@@ -262,6 +262,10 @@
       <Project>{eee9cd52-ca10-41ae-ac6e-98ae4bbdfa2d}</Project>
       <Name>CommonUxComponents</Name>
     </ProjectReference>
+    <ProjectReference Include="..\AccessibilityInsights.Extensions.AzureDevOps\Extensions.AzureDevOps.csproj">
+      <Project>{58dbf9fe-0029-408f-a8a4-23ae500e938d}</Project>
+      <Name>Extensions.AzureDevOps</Name>
```
Comment (jalkire) [inline src/AccessibilityInsights/AccessibilityInsights.csproj]: Don't think you wanted this here

Comment (karanbirsingh) [reply src/AccessibilityInsights/AccessibilityInsights.csproj]: you are correct, let me revert. thanks!

**Comment by karanbirsingh** [comment]
@DaveTryon is helping check win7 support

**Comment by DaveTryon** [comment]
I tested this on Win7. The result was what I expected--no image is displayed. That's consistent with what the main Ada-cat image does. Everything works, it just has no images whatsoever.

**Pull request merged**